src/main/java/org/galatea/starter/entrypoint/RestExceptionHandler.java

```diff
@@ -73,6 +74,15 @@
     return buildResponseEntity(error);
   }
 
+  @ExceptionHandler(FeignException.class)
+  protected ResponseEntity<Object> handleFeignException(
+      final FeignException exception) {
+    log.debug("Feign Exception Error Thrown",exception);
```

Comment:
Small detail but just for code style, there should always be a space after commas in a parameter list